src/components/timeline/Tooltip.svelte

```diff
@@ -80,6 +83,18 @@
     }
   }
 
+  export function reset() {
```

Comment:
Is this function used?

Reply:
Looks like this was added to support functionality in the timeline that once existed in `ExternalSourceManager` - removing as we've retired that timeline feature on ESM.